sdk/storage/azure-storage-blob/azure/storage/blob/_shared/downloads.py

```diff
@@ -383,32 +383,32 @@ def _initial_request(self):
 
         return response
 
-    def content_as_bytes(self, max_connections=1):
+    def content_as_bytes(self, max_concurrency=1):
```

Comment:
I guess moving this parameter to the constructor will be a separate PR specifically for the download stream API?

Reply:
yes, I'll do it here https://github.com/Azure/azure-sdk-for-python/issues/7504